.github/workflows/ci.yml

```
@@ -121,9 +121,9 @@ jobs:
       - name: Run tests
         id: test
         shell: msys2 {0}
-        working-directory: build/tests
+        working-directory: build
         run: |
-          ./test_libdisplaydevice.exe --gtest_color=yes
+          ctest --extra-verbose
```

After reviewing the output, I think I'd prefer to use the test binary in the ci.

```suggestion
          ./tests/test_libdisplaydevice --gtest_color=yes
```